mono/editor/monodevelop_instance.cpp

```diff
@@ -36,7 +36,7 @@ void MonoDevelopInstance::execute(const Vector<String> &p_files) {
 	MonoObject *exc = NULL;
 
 	Variant files = p_files;
-	const Variant *args[1]{ &files };
+	const Variant *args[1];{ &files; };
```

Comment:
That does not make sense. I assume it was supposed to be `const Variant *args[1] = { &files };` or something like it.

Reply:
Haha, yes this effectively does this

```cpp
const Variant *args[1]; // hey cool, it's an array
// let's open a scope
{
    &files; // well, this doesn't really do anything.
}
```

I love C/C++ grammar :)

Reply:
Made the changes requested.